### PR TITLE
feat: cookieless embed server cookies

### PR DESCRIPTION
**Pull request by bryans99**
Stores embed session data in the embed servers cookie. Cookie is encrypted and signed. Replaces in memory cache implementation. The cookie has the following advantage:
1. can be used in clustered environment.
2. survives server restart.

In addition embed routes used by the development server have been extracted to their own typescript file. This is in preparation for creating a tsnode implementation of the demo server.

### CODE REVIEW

File: webpack-devserver.config.js [hunk 1]
```diff
@@ -3,14 +3,7 @@ var config = require('./config')
...
 var user = require('./demo/demo_user.json')
-var { createSignedUrl } = require('./server_utils/auth_utils')
-var {
-  acquireEmbedSession,
-  generateEmbedTokens,
-  setConfig,
-} = require('./server_utils/cookieless_utils')
-
-setConfig(config)
+var {addRoutes} = require('./server_utils/routes')
```
Comment (jkaster) [inline webpack-devserver.config.js:6]: seems prettier or the linter will complain here?

**Review by jkaster — COMMENTED** [review]
Some questions

File: README.md [hunk 2]
```diff
@@ -227,14 +227,14 @@ Cookieless embed sessions are associated with the user's browser user agent. It
...
 - `api_token` - this token is used for api calls. This token lives for 10 minutes.
 
 A time to live for each token is also returned for each token. It is important that the response of the `acquire_embed_cookieless_session` be returned to the browser with the exception of the `session_reference_token`.
 
-The example below uses javascript, but you may utilize the Looker SDK in the language of your choice. Note that this is very simplistic implementation for demonstration purposes only. An actual implementation should be a lot more robust.
+The example below uses javascript, but you may utilize the Looker SDK in the language of your choice. Note that this is very simplistic implementation for demonstration purposes only. An actual implementation should be a lot more robust. The example below uses a simple in memory cache to keep track of the `session_reference_token`. An in memory cache will not work in a clustered environment so use a distributed cache such as `redis` for production implementations. Alternatively, the `session_reference_token` can be saved in an encrypted session cookie. The demo has been updated utilize session cookies instead of the in memory cache.
```
[the remark this answers is not in the document]
Comment (bryans99) [reply README.md]: Updated quite a bit. To be honest, I will probably be revisiting this throughout the week as I write more documention

File: webpack-devserver.config.js (hunk 1 above)
Comment (bryans99) [reply webpack-devserver.config.js]: js files are not linted or prettified. I've corrected manually.

Comment (jkaster) [reply webpack-devserver.config.js]: ah, right. sorry I missed the extension.

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**